flake.nix

```diff
@@ -106,6 +106,7 @@
               "-DBUILD_SHARED_LIBS=ON"
               "-DMANIFOLD_PAR=${if parallel then "ON" else "OFF"}"
             ];
+            doCheck = true;
```

Comment:
build_mxe still looks like it's not testing - also build_mac_..._subdir, unless that's intentional?

Reply:
mxe cannot be tested: we are building some windows exe on linux. at least we cannot test it without another job on windows...

bulid_mac_..._subdir... is just to test the cmake consumer, see if we can build when put inside a subdirectory in a larger project.